packages/utilities/gel-foundations/src/breakpoints.js

```diff
@@ -1,14 +1,6 @@
-/* 
-   Screen sizes for GEL Typography
-   These namings are based on the GEL description. They are also known as group A, group B and group D
-   Link to relevant GEL docs: http://www.bbc.co.uk/gel/guidelines/typography#type-sizes
-*/
-export const GEL_GROUP_A_MAX_WIDTH = 19.9375; // 319px
-export const GEL_GROUP_B_MIN_WIDTH = 20; // 320px
-export const GEL_GROUP_B_MAX_WIDTH = 37.4375; // 599px
-export const GEL_GROUP_CD_MIN_WIDTH = 37.5; // 600px
-
 /*
+    Breakpoints documentation: https://confluence.dev.bbc.co.uk/display/WSFrontPGS/Screen+sizes+and+Breakpoints
```

Comment:
Probably not useful to link to our internal confluence as external contributors/users won't be able to view it. Does it contain any extra info that isn't inferred from below?

Reply:
Good point. The confluence doc refers more explicitly to the const names that are in the file - ('Group A', 'Group B', 'Group C+D') so I thought it was more useful, but didn't consider its lack of accessibility for externals. Have removed!